ultra/ultra/config.yaml

```diff
@@ -43,6 +43,12 @@ tasks:
     hard:
       train: "scenarios/task1/train_hard*"
       test:  "scenarios/task1/test_hard*"
+    high-density:
+      train: "scenarios/task1/train_high-density*"
+      test:  "scenarios/task1/test_high-density*"
+    mid-density:
+      train: "scenarios/task1/train_mid-density*"
+      test:  "scenarios/task1/test_mid-density*"
```

Comment:
What are these extra levels for?

Reply:
This were for specific density based levels, we can remove them as well

Reply:
@christianjans I will remove them once this PR is ready to go. Those are only for debugging purposes